.github/workflows/ci.yml

```diff
@@ -185,23 +187,27 @@ jobs:
       fail-fast: false
       matrix:
         framework: [ "toga", "pyside6", "pygame", "console" ]
-        runner-os: [ "macos-12", "macos-14", "ubuntu-22.04", "windows-latest" ]
+        runner-os: [ "macos-13", "macos-latest", "ubuntu-latest", "windows-latest" ]
 
   verify-apps:
     name: Build app
     needs: [ package, package-automation, unit-tests ]
-    uses: beeware/.github/.github/workflows/app-build-verify.yml@main
+    # TODO: Revert to main when .github#172 is merged
+    uses: beeware/.github/.github/workflows/app-build-verify.yml@599f2fbb61c17aa94a962f26647f31a879a31052
```

Comment:
Do you want to confirm there won't be any surprises with `app-create-verify.yml@599f2fbb61c17aa94a962f26647f31a879a31052` as well?